server/routes/bitbucket/bitbucket_selection.py

```
@@ -420,6 +420,113 @@ def _store_hook_uuid(hook_uuid: str) -> None:
         return False
 
 
+@bitbucket_selection_bp.route("/repo-selections/change-gating", methods=["PUT"])
+@require_permission("connectors", "write")
+def update_change_gating_bulk(user_id):
```

Ruff (0.16.1)

[warning] Too many return statements (7 > 6)

(PLR0911)